dist/src/main/resources/application-auth-oidc.properties

```diff
@@ -1,8 +1,8 @@
-spring.security.oauth2.client.registration.oidcclient.client-id=oidcclient2
-spring.security.oauth2.client.registration.oidcclient.client-secret=secret2
-spring.security.oauth2.client.registration.oidcclient.authorization-grant-type=authorization_code
-spring.security.oauth2.client.registration.oidcclient.redirect-uri=http://localhost:8080/login/oauth2/code/oidcclient2
+spring.security.oauth2.client.registration.oidcclient.client-id=${camunda.security.auth.oidc.client-id:}
+spring.security.oauth2.client.registration.oidcclient.client-secret=${camunda.security.auth.oidc.client-secret:}
+spring.security.oauth2.client.registration.oidcclient.authorization-grant-type=${camunda.security.auth.oidc.grant-type:authorization_code}
+spring.security.oauth2.client.registration.oidcclient.redirect-uri=${camunda.security.auth.oidc.redirect-uri:}
 spring.security.oauth2.client.registration.oidcclient.provider=oidcclient
-spring.security.oauth2.client.registration.oidcclient.scope=openid,profile
+spring.security.oauth2.client.registration.oidcclient.scope=${camunda.security.auth.oidc.scopes:openid,profile}
 
-spring.security.oauth2.client.provider.oidcclient.issuer-uri=http://auth.identity:9000
+spring.security.oauth2.client.provider.oidcclient.issuer-uri=${camunda.security.auth.oidc.issuer-uri:}
```

Comment:
We do have `camunda.security.authentication.method` now. Should we use the longer namespace here too? Also, do these properties need to be reflected in the configuration classes?